(node:1801566) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
(Use `node --trace-warnings ...` to show where the warning was created)
Started with the following options:
{
  debug: true,
  mitmIp: '10.0.3.1',
  autoAccess: true,
  autoAccessCache: 5000,
  maxAttemptsPerConnection: 6,
  sshServerIdentifier: 'SSH-2.0-OpenSSH_6.6.1p1 Ubuntu-2ubuntu2',
  containerMountPathPrefix: '/var/lib/lxc',
  containerMountPathSuffix: 'rootfs/',
  loggingAttackerStreams: '/home/student/MITM/logs/session_streams',
  loggingAuthenticationAttempts: '/home/student/MITM/logs/authentication_attempts',
  loggingLogins: '/home/student/MITM/logs/logins',
  loggingLogouts: '/home/student/MITM/logs/logouts',
  loggingKeystrokes: '/home/student/MITM/logs/keystrokes',
  containerName: 'nobanner-nosession-41550',
  containerIp: '10.0.3.84',
  mitmPort: 41550,
  autoAccessFixed: 1
}
2024-11-07 12:58:00.969 - [Info] MITM Version: 2
2024-11-07 12:58:00.971 - [Info] Auto Access Enabled: true
2024-11-07 12:58:00.983 - [Debug] [Init] Auto Access Theshold Achieved: false
2024-11-07 12:58:01.034 - [Info] SSH man-in-the-middle server for 10.0.3.84 listening on 10.0.3.1:41550
2024-11-07 12:59:00.209 - [Debug] [Connection] Attacker connected: 47.245.92.217 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-07 12:59:01.714 - [Debug] [Auth] Attacker 47.245.92.217 trying to authenticate with 'password'
2024-11-07 12:59:01.716 - [Debug] [Auto Access] Attacker: 47.245.92.217, Threshold: 1, Attempts: 1
2024-11-07 12:59:01.716 - [Debug] [Auto Access] Compromising the honeypot
2024-11-07 12:59:01.716 - [Debug] [Auto Access] Adding the following credentials: 'pontevedra:pontevedra'
2024-11-07 12:59:01.990 - [Debug] [Auto Access] Auto-access is now disabled for the remainder of this MITM server instance
2024-11-07 12:59:01.992 - [Debug] [Auth] Valid credentials - Password Authentication
2024-11-07 12:59:01.992 - [Debug] [LXC] Attempting to connect to the honeypot: 10.0.3.84
2024-11-07 12:59:02.074 - [Debug] [LXC-Auth] Attacker authenticated and is inside container
2024-11-07 12:59:03.126 - [Debug] [EXEC] Noninteractive mode attacker command: cd ~; chattr -ia .ssh; lockr -ia .ssh
2024-11-07 12:59:04.673 - [Debug] [Connection] Attacker connected: 47.245.92.217 | Client Identification: SSH-2.0-libssh_0.9.6
2024-11-07 12:59:06.169 - [Debug] [Auth] Attacker 47.245.92.217 trying to authenticate with 'password'
2024-11-07 12:59:06.170 - [Debug] [Auth] Attacker authentication error: Invalid credentials - User does not exist
2024-11-07 12:59:06.172 - [Debug] [Auth] Attacker: 47.245.92.217 has so far made 1 attempts. Remaining: 5 attempts
2024-11-07 12:59:06.693 - [Debug] [Connection] Attacker closed the connection
2024-11-07 12:59:07.929 - [Debug] [Connection] Attacker connected: 47.245.92.217 | Client Identification: SSH-2.0-libssh_0.9.6
{"level":"error","message":"Forever detected script was killed by signal: SIGKILL"}
